stablehlo/tools/StablehloOptMain.cpp

```diff
@@ -38,6 +39,7 @@ int main(int argc, char **argv) {
   mlir::registerAllDialects(registry);
   mlir::registerAllExtensions(registry);
   mlir::stablehlo::registerAllDialects(registry);
+  registry.insert<mlir::stablehlo::check::CheckDialect>();
```

Comment:
I'm even more OK with this now that I see that `opt` has been able to handle InterpreterDialect this entire time. Check is effectively a feature of the interpreter currently.